mantidproject/mantid
# Switch macOS system test runner to use Python from MantidWorkbench bundle

**Description of work.**

When MantidPlot is disabled the `MantidPlot.app` bundle no longer exists and the system tests fail to even start. This switches to use the Python from inside the `MantidWorkbench.app` bundle.

**To test:**

I tested this as follows:

* Run `cmake -DENABLE_MANTIDPLOT=OFF -DENABLE_CPACK=ON -DCPACK_PACKAGE_SUFFIX=` and build
* Build a package with `ninja package`
* Change directory to the root of the cloned source code
* `mkdir build` and move the package generated from step 2 to `build`
* Run `WORKSPACE=$PWD EXTRA_ARGS="-R CodeConventions" bash buildconfig/Jenkins/systemtests` and the test should run and pass 

Refs #28650

*This does not require release notes* because **it is an internal task.**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
